chitchat/src/server.rs

```diff
@@ -37,11 +37,13 @@ async fn dns_refresh_loop(
     seed_addrs_tx: watch::Sender<HashSet<SocketAddr>>,
 ) {
     let mut interval = time::interval(DNS_POLLING_DURATION);
-    // We actually do not want to run the polling loop right away,
+    // We actually DO want to run the polling loop right away,
```

Comment:
Ok the comment sounds strange :)

We do want the polling but we made the first one already before.

The comment could be 

```
The caller has just run the dns resolution. We can wait one tick before retrying.   
```

Reply:
Also, now there are two ticks. Is this what you wanted?